dashboard/common/db_util.py

```diff
@@ -36,26 +36,31 @@ def get_course_name_from_id(course_id):
     :rtype: [str]
     """
     logger.debug(get_course_name_from_id.__name__)
-    course_id = canvas_id_to_incremented_id(course_id)
+    warehouse_id = canvas_id_to_incremented_id(course_id)
     course_name = ""
     if (course_id):
         with django.db.connection.cursor() as cursor:
-            cursor.execute("SELECT name FROM course WHERE id = %s", [course_id])
+            cursor.execute("SELECT name FROM course WHERE warehouse_id = %s", [warehouse_id])
             row = cursor.fetchone()
             if (row != None):
                 course_name = row[0]
     return course_name
 
 
-def get_course_view_options (course_id):
-
+def get_course_view_options(course_id):
```

Comment:
This function seems to be no longer in use. I turned up no other occurrences through a search, and in the primary place where it would be used (see `get_course_info` method in `views.py`, line 120), a QuerySet is used. Should we remove it?